modules/administration/pages/maintenance-windows.adoc

```diff
@@ -0,0 +1,25 @@
+[[maintenance-windows]]
+= Maintenance Windows
+
+The maintenance windows feature in {productname} allows you to schedule actions to occur during your maintenance window period ahead of time, and to create repeating tasks.
+
```

Comment:
I think that it'd be good, if we clarified, what the terms *Calendar* and *Schedule* are. These are some facts that should describe them:

- *Calendar* is basically an [iCalendar](https://en.wikipedia.org/wiki/ICalendar) file imported to SUMA
- *Schedule* uses events defined in *Calendar* to define/generate maintenance windows
- *Schedule* can be assigned to *System*s
- A *System* with assigned *Schedule* can't run "restricted" action outside of maintenance windows
- Restricted actions are actions, that modify the system significantly (and could potentially break it) - e.g. package installation, upgrade, service pack migration, highstate application. Unrestricted actions are fairly safe actions - e.g. package profile update, hardware refresh, subscribing to software channels.

![diagram-schedule-1](https://user-images.githubusercontent.com/1412268/87919495-2989bb80-ca78-11ea-8114-414054585a1e.png)